### PR TITLE
Julio/graphql test

### PR DESCRIPTION
**Pull request by hoolioh**
## Description

Add tests for GraphQL monitoring. This tests assert that a rule set on the address `graphql.server.all_resolvers` triggers an attack.

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com
* [ ] if a scenario is added (or removed), add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

File: tests/appsec/test_blocking_addresses.py [hunk 1]
```diff
@@ -722,3 +723,60 @@ def test_non_blocking(self):
     def test_blocking_before(self):
         """Test that blocked requests are blocked before being processed"""
         # TODO
+
+
+@scenarios.appsec_blocking
+@coverage.good
+@released(
```
Comment (cbeauchesne) [inline tests/appsec/test_blocking_addresses.py:730]: Could you use the new manifest files for this please ?

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: tests/appsec/test_blocking_addresses.py (hunk 1 above)
Comment (hoolioh) [reply tests/appsec/test_blocking_addresses.py]: sure

File: tests/appsec/test_blocking_addresses.py [hunk 2]
```diff
@@ -722,3 +723,57 @@ def test_non_blocking(self):
     def test_blocking_before(self):
         """Test that blocked requests are blocked before being processed"""
         # TODO
+
+
+@scenarios.appsec_blocking
+@coverage.good
+class Test_Blocking_graphql_resolvers:
```
Comment (cbeauchesne) [inline tests/appsec/test_blocking_addresses.py:730]:
```suggestion
class Test_BlockingGraphqlResolvers:
```

**Review by cbeauchesne — APPROVED** [review]
Just a small python style request change (do not forget the manifest), and all good to me!

**Pull request merged**